client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/disabled-badge/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Badge } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import Tooltip from 'calypso/components/tooltip';
+
+import './style.scss';
+
+export default function DisabledBadge() {
```

Comment:
Thanks for suggesting this, @yashwin, I attempted to create a reusable badge out of this, and I ended up complicating things. 

My idea was to create a Hoverable badge with the flexibility to swap popovers without making the component aware of the exact popover. It becomes very complicated, especially with passing the ref context outside the consuming component. It becomes harder to consume w/ having to extract the ref context outside the badge component. 

We could solve this by moving the ref logic outside the badge context, but it will only leave us w/ a styled badge from the `@automattic/components` package, and I thought it wasn't worth it.